bokehjs/gulp/tasks/scripts.coffee

```diff
@@ -62,6 +62,7 @@ gulp.task "scripts:tsjs", ["scripts:coffee", "scripts:js", "scripts:ts"], () ->
         gutil.log("#{gutil.colors.red(real)}#{rest}")
         return
 
+      # XXX: can't enable "6133", because CS generates faulty code for closures
```

Comment:
This is annoying and significantly weakens this PR, but there's nothing we can do about this. Given code like this:
```
arr = [0...10]
```
CS generates code like this:
```
var arr, i, ref1;
arr = function() {
  var i, ref1;
}
```
(I skipped irrelevant code) which has obvious unused variables outside the closure.